app/unitPages.rb

```diff
@@ -1263,6 +1265,7 @@ def rebuildIndirectLinks(unitID, hier = nil)
     attrs = {}
     inAttrs[:title] and attrs[:title] = sanitizeHTML(inAttrs[:title])
     inAttrs[:html]  and attrs[:html]  = sanitizeHTML(inAttrs[:html])
+    inAttrs[:twitter_handle]  and attrs[:twitter_handle]  = sanitizeHTML(inAttrs[:twitter_handle])
```

Comment:
```suggestion
    inAttrs[:twitter_handle]  and attrs[:twitter_handle]  = inAttrs[:twitter_handle]
```
We don't render the twitter handle as HTML do we? It gets rendered as plain text, so no need to sanitize it. That said, it's probably harmless to do so.